### PR TITLE
Browser Adapter

### PR DESCRIPTION
**Pull request by banderson623**
Initial stab at a browser adapter.

Also includes the formalized report response function

### CODE REVIEW

File: src/adapters/browser.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    resolve([
+      response('Viewport', viewportWidth() + "x" + viewportHeight()),
+      response('url', url()),
+      response('Browser', browser()),
+    ]);
+  });
+}
+
+const browserAdapter = {
+  discover,
```
Comment (trinhathu) [inline src/adapters/browser.js]: Where are `discover` and `load` coming from?

File: src/utils/response.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+// Formalizing the response type.
+export default function response(label, value) {
+  return {
+    label: label,
+    value: value,
+  }
+}
```
Comment (emilymalecbrown) [inline src/utils/response.js]: semicolon

File: src/utils/response.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+// Formalizing the response type.
+export default function response(label, value) {
+  return {
+    label: label,
+    value: value,
+  }
```
Comment (emilymalecbrown) [inline src/utils/response.js]: semicolon 😝

File: src/utils/response.js (hunk 2 above)
Comment (banderson623) [reply src/utils/response.js]: doh, thank you.

File: src/adapters/browser.js (hunk 1 above)
Comment (banderson623) [reply src/adapters/browser.js]: WHOOOOPS too many name changes. fixing :)

**Pull request merged**